app/views/pages/instances/create.liquid

```diff
@@ -0,0 +1,11 @@
+---
+method: post
+slug: instances/create
+---
+
+{% liquid
+    include 'require_auth'
+
+    function _ = 'lib/commands/instances/create', domain: context.params.domain, pos_cli_token: context.params.pos_cli_token, available: true
```

Comment:
nice! what happens though if there are any validation errors - for example someone has not provided token or domain is not a valid domain?

what you want is to have an `if` statement and redirect only if the result of instances/create is valid, otherwise re-render the form with validation errors